tests/test_playbooks/fixtures/katello_smart_proxy-4.yml

```diff
@@ -105,7 +105,7 @@ interactions:
       x-xss-protection: [1; mode=block]
     status: {code: 200, message: OK}
 - request:
-    body: !!python/unicode '{"smart_proxy": {"download_policy": "background"}}'
+    body: !!python/unicode '{"smart_proxy": {"download_policy": "inherit"}}'
```

Comment:
This is cheating.

Reply:
I literally didn't know what I was doing and how all those tests work. After changing it in the code the tests failed. So I cheated.

If you tell me how it's done correctly, I'll change it. Or you create a new pull request the right way. It's a small change anyway...

Reply:
Sorry for being too brief. These fixtures are actual server responses, and by hand changing them we would loose trust in them. So they should only be recreated by rerecording (see the readme).

But there is also a deeper issue with the change here. I do not think that the modules are supposed to _only_ support Katello >= 4.0. So we cannot just remove the old option. At the very least, you should keep it and tell about the version dependence in the modules inline documentation. Maybe some version dependent validation on execution is also possible.

Last but not least, this change needs a changelog snippet.

Reply:
Well, I guess that's beyond my capabilities and available time at the moment. I thought it was a straight-forward change but I didn't see everything else affected. I'll close this and let this to someone more experienced...